src/coreclr/System.Private.CoreLib/src/System/Reflection/Emit/DynamicMethod.CoreCLR.cs

```diff
@@ -172,7 +172,9 @@ Signature LazyCreateSignature()
                     Span<ParameterCopyBackAction> shouldCopyBackParameters = new(ref argStorage._copyBack0, argCount);
 
                     StackAllocatedByRefs byrefStorage = default;
+                    #pragma warning disable CS8500
```

Comment:
What is CS8500?  I just want to understand what the compiler is concerned about.  I'm surprised taking the address of anything would produce a warning, regardless of where it came from.

Reply:
`StackAllocatedByRefs` is a managed type by virtue of having `ref` fields. The warning here is fallout from the LDM decision of downgrading memory safety issues from errors to warnings inside an unsafe block. Taking the address of a managed type has traditionally been an error but now it's a warning when done inside unsafe blocks.

Reply:
Aha. Ok.